cmd/kubeapps-apis/Dockerfile

```diff
@@ -3,12 +3,12 @@
 
 # syntax = docker/dockerfile:1
 
-FROM bitnami/golang:1.17.6 as builder
+FROM bitnami/golang:1.18.2 as builder
 WORKDIR /go/src/github.com/vmware-tanzu/kubeapps
 COPY go.mod go.sum ./
 ARG VERSION="devel"
 
-ARG BUF_VERSION="1.0.0-rc11"
```

Comment:
When I saw `1.0.0-rc11` I thought "wow, this must be veeery old". But it is only [4 months old](https://github.com/bufbuild/buf/releases/tag/v1.0.0-rc11) 😄 .
The buf project is on 🔥